packages/sources/svelte/src/components/overlays/VtmnModal/VtmnModal.svelte

```diff
@@ -67,9 +67,6 @@
             <slot name="description" />
           </p>
         {/if}
-        {#if $$slots.actions}
```

Comment:
Why? Does this choice come from Vitamin Design core team?

Reply:
@GaspardMathon okay thanks. Why didn't we remove it completely for all technologies ?

Reply:
@lauthieb It was already absent in the other technologies.